style.css

```diff
@@ -129,10 +171,34 @@ footer {
 
 
 /***** media *****/
-@media (max-width: 665px) {
+@media (max-width: 750px) {
+    .winner {
+        box-shadow: inset 0 5px 1px rgb(196, 196, 224), 0 0 1px 1.5rem rgba(255, 255, 255, 0.1), 0 0 1px 3rem rgba(255, 255, 255, 0.1), 0 0 1px 4.5rem rgba(255, 255, 255, 0.1);
```

Comment:
I would save rgba(255, 255, 255, 0.1) and other identical colors in a variable, so if you need to change this you will only change the variable value.

Reply:
Fixed.